### PR TITLE
Overhaul UI

### LINKED ISSUE #26: UI Overhaul (Parent Task)
Make a separate branch for all of the UI related work we will be working on. 

Our goal is to get a UI that is pretty similar to what Uppy has made. It is an open source project so you could easily look into their code for inspiration on how they stylized or designed certain stuff on the UI: https://github.com/transloadit/uppy

You could see how their package works in here:
https://uppy.io/examples/

### PR DESCRIPTION
**Pull request by MohamedBechirMejri**
Fixes #26 #27

### Task List

- [X] Clone Uppy's UI
- [X] Add Better Animations
- [X] Responsive UI
- [x] Test if Upload Works
- [x] Test if grabbing files from the Cloud Works
- [x] Test if tailwind works in production
- [x] Add Drag and Drop Feature
- [ ] Improve Docs
- [ ] Add Image Editor

### CODE REVIEW

**Comment by MohamedBechirMejri** [comment]
Preview: 

https://github.com/uNotesOfficial/upup/assets/76930306/53e5e655-928b-4e21-a495-b916b8d566df

File: .storybook/main.ts [hunk 1]
```diff
@@ -8,13 +8,23 @@ const config: StorybookConfig = {
...
+            name: '@storybook/addon-styling',
             options: {
-                mdxPluginOptions: {
-                    mdxCompileOptions: {
-                        remarkPlugins: [remarkGfm],
-                    },
+                // Check out https://github.com/storybookjs/addon-styling/blob/main/docs/api.md
+                // For more details on this addon's options.
+                postCss: {
+                    implementation: require.resolve('postcss'),
```
Comment (bassem97) [inline .storybook/main.ts:27]: nicely done !

**Review by bassem97 — APPROVED** [review]
IMPRESSIVE !

**Comment by MohamedBechirMejri** [comment]
> IMPRESSIVE !

TYSM

**Comment by arijhajji-1** [comment]
AMAZING @MohamedBechirMejri  !

**Comment by MohamedBechirMejri** [comment]
> AMAZING @MohamedBechirMejri  !

Thx

**Pull request merged**